libr/anal/p/anal_arm_cs.c

```diff
@@ -4049,7 +4049,7 @@ jmp $$ + 4 + ( [delta] * 2 )
 		op->type = R_ANAL_OP_TYPE_PUSH;
 // 0x00008160    04202de5     str r2, [sp, -4]!
 // 0x000082a0    28000be5     str r0, [fp, -0x28]
-		if (REGBASE(1) == ARM_REG_FP) {
+		if ((insn->id != ARM_INS_PUSH) && (REGBASE(1) == ARM_REG_FP)) {
```

Comment:
i think its cleaner to remove the fallthrough case above and duplicate the code so the if can be removed here. it will be easier to read and easier to maintain. Can you do all those changes? if not i can handle that. thanks for submiting the PR, code changes are much easier to review like this

Reply:
Ok, clear.
Should I proceed with this PR or create a new one? Sorry, I've made too many commits for such a small fix :)
And yes, please feel free to update this PR as you need it, because I'm not so familiar with the code and with the process. I would appreciate your cooperation.

Reply:
yep, do the change in this PR, sorry for the delay i was buried in meetings

Reply:
i'll merge after that change :)

Reply:
That's fine :) I've just updated it. Could you please have a look once again when you get a minute?